modules/hcp-dr-oadp-dpa.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-disaster-recovery-oadp-auto.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="hcp-dr-oadp-dpa_{context}"]
+= Automating the backup and restore process by using a DPA
+
+You can automate parts of the backup and restore process by using a Data Protection Application (DPA). The DPA defines information including backup locations and Velero pod configurations.
```

Comment:
> automate parts of the backup and restore process

Which parts can be automated?  And, what do I do after creating the DPA object?

Reply:
Good questions. Will add more details.